plug.vim

```diff
@@ -1332,6 +1698,8 @@ function! s:upgrade()
       endif
     elseif s:ruby
       call s:upgrade_using_ruby(new)
+    elseif s:py2
+      call s:upgrade_using_python(new)
     else
       return s:err('curl executable or ruby support not found')
```

Comment:
I did a comment on a line but I don't know, since it wasn't a line you added, if github attached my comment on that line or at the file instead.